packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@t3n/components",
-  "version": "0.5.17",
+  "version": "0.5.16",
```

Comment:
Donwgrade?

Reply:
Nein, Birte hatte in der fehlerhaften Version versucht ein Release zu machen, das zwar im Projekt und Github die Version geraised, beim publish zu NPM jedoch gefailed ist. Habe das Tag in Github gelöscht und die Versionsnummer in der Package.json gefixed.